docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -397,7 +397,7 @@ public:
     name getWinner(const game &currentGame);
 ```
 
-The complete tictactoe.hpp file can be downloaded from github here: [Tic-tac-toe tutorial hpp source.](https://github.com/EOSIO/welcome/tree/master/docs/03_tutorials/src/tictactoe/tictactoe.hpp "tic-tac-toe example hpp")  
+The complete tictactoe.hpp file can be downloaded from github here: [Tic-tac-toe tutorial hpp source.](https://github.com/EOSIO/welcome/blob/release/2.0.x/src/tictactoe/tictactoe.hpp  "tic-tac-toe example hpp")
```

Comment:
another option is to link to the version which is in master branch

Reply:
Ok adding option d, which is somewhat similar to option a:

  d. `master`: if we update the tictactoe sources in `develop`, the link will point to potentially outdated content (unless `develop` gets merged to `master` after each update, which is not good practice). When merged into `release/2.1.x` we'll still need to update the link to `release/2.1.x`.

I will approve the PR for now since the changes are not breaking, but we need to decide which option to adopt (a,b,c,d) when merging content to `develop` that contains links to `release/x.y.z` resources.